6_QAQC/src/QAQC_Approach3_lat_long_doy_bins.R

```diff
@@ -0,0 +1,70 @@
+library(scipiper)
+library(dplyr)
+library(plotly)
+library(tidyverse)
+set.seed
+
+#getting the data from google drive and changing the file to RDS file without ind
+gd_get("5_data_munge/out/daily_temperatures.rds.ind")
+# to get the sites longitude and latitude data from google drive,
+gd_get("5_data_munge/out/all_sites.rds.ind")
+
+#creating an r object for the daily temp data
+daily_dat <- readRDS('5_data_munge/out/daily_temperatures.rds') %>%
```

Comment:
Can you rewrite the scipiper steps so that it's ready to be put in the pipeline?

e.g., `gd_ge`t to `sc_retrieve` and so on.